pages/take3.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: The Web — Take 3
+permalink: take3
+---
+
+The original web, call it web 1, allowed people if they were technically capable to make their own web sites, to blog and to link from their own work to other work they valued. (This used URLs, HTTP and HTML).   
+
+With web 2, specific web sites, using the web app technology (JavaScript), provided social and interactive functionality to users, so they could interact together and  in groups. This relied on the social network sites (SNS) storing all the data about all of the people and creating new value and insight -- for the user but also for ad systems which ran the companies revenue stream.  "Web 2.0" as some people call it is based on user-generated content. 
+
+Solid, as a new wave, we like to call the web – take 3.   It can be thought of as mid-course correction for the web.  It does add more power, but also questions and throttles back the emphasis on the only revenue model for the web being ads, and flips the privacy world right-side up.
+
+The original goal of the web was to be a place for people to be able to work really well together. To be able to be creative together ("Intercreativity") and to be able to collaborate really well -- and now I would add compassion at a distance.  It is maybe not surprising that this didn't happen when you think of the fact that we did not have a common identity, so we could not do simple global access control.   In fact most collaborate work is typically restricted to  a few people at first, and then released to more and more, th elact of access control was a clear problem.  Of course the social networks solved the problem within their own systems with their own specific identity and access control, but these were not standard, not interoperable, and meant you hadto have your entire life and work within one silo for it to work.
```

Comment:
```suggestion
The original goal of the web was to be a place for people to be able to work really well together. To be able to be creative together ("Intercreativity") and to be able to collaborate really well -- and now I would add compassion at a distance.  It is maybe not surprising that this didn't happen when you think of the fact that we did not have a common identity, so we could not do simple global access control. In fact most collaborative work is typically restricted to a few people at first, and then released to more and more, the lack of access control was a clear problem. Of course the social networks solved the problem within their own systems with their own specific identity and access control, but these were not standard, not interoperable, and meant you had to have your entire life and work within one silo for it to work.

-- Tim Berners Lee
```
Since this paragraph contains "I", I think it should be signed using Tim's name, which incidentally gives plenty of legitimacy to describe the original goal of the web.